hashedindex/textparser.py

```diff
@@ -29,7 +29,7 @@ class InvalidStemmerException(Exception):
 _punctuation = _punctuation.replace('-', '')
 
 _re_punctuation = re.compile('[%s]' % re.escape(_punctuation))
-_re_token = re.compile(r'[A-z0-9]+')
+_re_token = re.compile(r'[A-z0-9]+|\s+')
```

Comment:
Getting this correct in combination with #16 is tricky, and worth being very careful about since there'll be a mege conflict to resolve.

Reply:
To prepare for this, I've applied some of the same regular expression & punctuation class naming conventions as #16.  It's not perfect but should reduce the potential for mistakes at conflict-resolution-time.